scipy/signal/tests/test_signaltools.py

```diff
@@ -109,6 +109,13 @@ def test_input_swapping(self):
         assert_array_equal(convolve(big, small, 'valid'),
                            out_array[1:3, 1:3, 1:3])
 
+    def test_convolve_method(self):
+        x = np.random.rand(10)
```

Comment:
I usually use `rng = np.random.RandomState(1234)` object and then
`rng.rand(...)` for this purpose. Feels a little cleaner than modifying a
global, but both should accomplish the same thing so not a big difference.